tests/integration_tests/README.md

```diff
@@ -1,52 +1,101 @@
-    * [bin/jq](https://stedolan.github.io/jq/)
-    * [bin/minio](https://github.com/minio/minio)
+1. The following executables must be copied or generated or linked into these locations, `sync_diff_inspector` can be
+   downloaded
+   from [tidb-community-toolkit](https://download.pingcap.org/tidb-community-toolkit-v4.0.2-linux-amd64.tar.gz)
+   , `tidb-server` related binaries can be downloaded
+   from [tidb-community-server](https://download.pingcap.org/tidb-community-server-v4.0.2-linux-amd64.tar.gz):
 
-    > If you are running tests on MacOS, tidb related binaries can be downloaded from tiup mirrors, such as https://tiup-mirrors.pingcap.com/tidb-v4.0.2-darwin-amd64.tar.gz. And `sync_diff_inspector` can be compiled by yourself from source [tidb-tools](https://github.com/pingcap/tidb-tools)
+   * `bin/tidb-server` # version >= 4.0.0-rc.1
+   * `bin/tikv-server` # version >= 4.0.0-rc.1
+   * `bin/pd-server`   # version >= 4.0.0-rc.1
+   * `bin/pd-ctl`      # version >= 4.0.0-rc.1
+   * `bin/tiflash`     # needs tiflash binary and some necessary so files
+   * `bin/sync_diff_inspector`
+   * [bin/go-ycsb](https://github.com/pingcap/go-ycsb)
+   * [bin/etcdctl](https://github.com/etcd-io/etcd/tree/master/etcdctl)
+   * [bin/jq](https://stedolan.github.io/jq/)
+   * [bin/minio](https://github.com/minio/minio)
 
-    > All Tiflash required files can be found in [tidb-community-server](https://download.pingcap.org/tidb-community-server-v4.0.2-linux-amd64.tar.gz) packages. You should put `flash_cluster_manager`, `libtiflash_proxy.so` and `tiflash` into `bin` directory in TiCDC code base.
+   > If you are running tests on MacOS, tidb related binaries can be downloaded from tiup mirrors, such as https://tiup-mirrors.pingcap.com/tidb-v4.0.2-darwin-amd64.tar.gz. And `sync_diff_inspector` can be compiled by yourself from source [tidb-tools](https://github.com/pingcap/tidb-tools)
+
+   > All Tiflash required files can be found in [tidb-community-server](https://download.pingcap.org/tidb-community-server-v4.0.2-linux-amd64.tar.gz) packages. You should put `flash_cluster_manager`, `libtiflash_proxy.so` and `tiflash` into `bin` directory in TiCDC code base.
 
 2. The following programs must be installed:
 
-    * `mysql` (the MySQL cli client, currently [mysql client 8.0 is not supported](https://github.com/pingcap/tidb/issues/14021))
+   * `mysql` (the MySQL cli client,
+     currently [mysql client 8.0 is not supported](https://github.com/pingcap/tidb/issues/14021))
+
+3. The user used to execute the tests must have permission to create the folder /tmp/tidb_cdc_test. All test artifacts
+   will be written into this folder.
+
+### Run integration tests in docker
+
+The following programs must be installed:
 
-3. The user used to execute the tests must have permission to create the folder /tmp/tidb_cdc_test. All test artifacts will be written into this folder.
+* [docker](https://www.docker.com/)
```

Comment:
Same as mysql client, is there any quick way to install these dependency by one step?

Reply:
I used the official documentation link.